docs/buildah-rmi.md

```diff
@@ -0,0 +1,29 @@
+## buildah-rmi "March 2017"
+
+## NAME
+buildah rmi - Removes an image or images. 
```

Comment:
Changed the title for rmi and have added the bit about failure on multiple images.  WIll add the same to rm.